src/data/storedEvals.js

```diff
@@ -0,0 +1,69 @@
+function getSubmittedData() {
+
+    const inputsNewForm = document.querySelectorAll('.candidateDetails');
+    const textAreas = document.querySelectorAll('textarea');
+    const radioValues = document.querySelectorAll('.radio-test');
+    const selectInputs = document.querySelectorAll('select');
+
+    inputsData = [];
+    radioData = [];
+    textareaData = [];
+    selectsData = [];
+
+    inputsNewForm.forEach(function (inputs) {
+        inputsData.push({
+            name: inputs.name,
+            value: inputs.value
+        });
+    })
+
+    radioValues.forEach(function (rad) {
+        if (rad.checked) {
+            radioData.push({
+                name: rad.name,
+                value: rad.value
+            });
+        }
+    })
+
+    textAreas.forEach(function (inputs) {
+        textareaData.push({
+            name: inputs.name,
+            value: inputs.value
+        });
+    })
+
+    selectInputs.forEach(function (inputs) {
+        selectsData.push({
+            name: inputs.name,
+            value: inputs.value
+        });
+    })
+
+    const savedObject = {
+        candidate: inputsData,
+        radio: radioData,
+        textarea: textareaData,
+        dropDown: selectsData
+    }
+
+    var currentObj = localStorage.getItem("Evaluations");
+
+    if (JSON.parse(currentObj) === null) {
+        let arr = [];
+        arr.push(savedObject);
+        localStorage.setItem(`Evaluations`, JSON.stringify(arr));
+  }else{
+        let arr = JSON.parse(currentObj);
```

Comment:
Hmm. This should be optimized to do the parsing only once. If we already have a couple thousand items to parse, there could be a noticeable performance degradation as is.

Reply:
Also, the code for both `if` and `else` branches is remarkably similar, with only the very first line of code being different, so we could clean it up a bit to keep it DRY.